chromium_src/chrome/browser/resources/extensions/item_list.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { ExtensionsItemListElement } from './item_list-chromium.js'
+
+(ExtensionsItemListElement.prototype as any).shouldShowEmptyItemsMessage_ = () => true;
```

Comment:
we need to come up with some test approach for such changes. Right now testing settings is not that simple because of shadow dom everywhere. I was thinking about doing something simple like:
* throwing if the thing we try to override is not there (instead of `console.log` as I see sometimes)
* add a test to open settings page and expect there's no exceptions

because with time these overrides will become obsolete and no one will notice this until a bug report from a user or QA.

Reply:
Could we possibly take advantage of the type of testing upstream does here with Mocha? https://source.chromium.org/chromium/chromium/src/+/main:chrome/test/data/webui/settings/settings_browsertest.cc